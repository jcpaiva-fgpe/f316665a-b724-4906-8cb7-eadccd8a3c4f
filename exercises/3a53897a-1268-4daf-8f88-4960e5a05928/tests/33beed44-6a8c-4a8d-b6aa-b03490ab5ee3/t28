12780 20542 12788 20548
3
-4 4
0 6
-8 8
