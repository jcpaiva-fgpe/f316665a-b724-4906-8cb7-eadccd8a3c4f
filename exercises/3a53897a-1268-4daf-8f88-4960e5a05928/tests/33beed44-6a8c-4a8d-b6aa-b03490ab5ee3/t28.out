1
Atenas 2
